src/main/java/org/candlepin/model/OwnerContentCurator.java

```diff
@@ -351,6 +371,10 @@ public int clearContentForOwner(Owner owner) {
             TypedQuery<Content> query = this.getEntityManager()
                 .createQuery(jpql, Content.class);
 
+            if (lockMode != null && !this.getDatabaseDialect().contains("hsql")) {
```

Comment:
It's unfortunate that we have to check for hsql here, but since we are, this warrants a good comment that explains why it's done

Reply:
Done.